### ISSUE TITLE
The results are not good on other Mujoco environments

### ISSUE THREAD

**Issue by weifan404** [ku2482/gail-airl-ppo.pytorch#2]
Hi, thanks for your wonderful project! It's easy to understand. And I got a good result in hopper.But when I transform it to the other environments like ant or humanoid, it got a bad work.So what parameters should I tune to get a better reward on these environment? Do you have any try?  Looking for your reply.Thanks!

**Comment by toshikwa** [comment]
Sorry for the late response.

In the gail paper, experiments are done with 25M training steps for the most environments including half cheetah, and 75M steps for humanoid.
I don't have enough computational resources to test on humanoid env, but at least I have tested on half cheetah with 20M steps and it worked well.

Have you tried on humanoid with 75M steps?
